core/controllers/base.py

```diff
@@ -390,7 +390,6 @@ def validate_and_normalize_args(self):
             else:
                 request_arg_keys.append(arg)
                 handler_args[arg] = self.request.get(arg)
-
```

Comment:
I don't think you need to make changes in this file -- please check the "Files Changed" tab and remove any changes that aren't necessary.